frontend/src/pages/Graphing/components/FunnelChart.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, Text } from '@highlight-run/ui/components'
+import * as style from '@pages/Graphing/components/Graph.css'
+import {
+	Funnel,
+	FunnelChart as RechartsFunnelChart,
+	LabelList,
+	LabelProps,
+	ResponsiveContainer,
+	Tooltip,
+} from 'recharts'
+
+import {
+	AxisConfig,
+	getColor,
+	InnerChartProps,
+	SeriesInfo,
+} from '@/pages/Graphing/components/Graph'
+import { FunnelDisplay } from '@pages/Graphing/components/types'
+
+const getCustomLabel = () => (props: LabelProps) => {
+	return (
+		<Box cssClass={style.labelWrapper}>
+			<Text
+				size="xxSmall"
+				weight="medium"
+				color="default"
+				cssClass={style.labelText}
+			>
+				{props.name}
+			</Text>
+		</Box>
+	)
+}
+
+const getCustomTooltip =
+	(data: any[] | undefined) =>
+	({ payload }: any) => {
+		if (!data?.at(0)) return null
+		const initialValue = data[0][data[0].Group]
+		return (
+			<Box cssClass={style.tooltipWrapper}>
+				{payload.map((p: any, idx: number) => (
+					<Box
+						display="flex"
+						flexDirection="column"
+						justifyContent="center"
+						alignItems="center"
+						gap="16"
+						padding="8"
+						key={idx}
+					>
+						<Text
+							size="small"
+							weight="medium"
+							color="default"
+							cssClass={style.tooltipText}
+						>
+							{p.name || '<empty>'}
+						</Text>
+						<Text
+							size="small"
+							weight="bold"
+							color="default"
+							cssClass={style.tooltipText}
+						>
+							{p.value + ' sessions: '}
+							{((100 * p.value) / initialValue).toFixed(1)}%
+						</Text>
+					</Box>
+				))}
+			</Box>
+		)
+	}
+
+export type FunnelChartConfig = {
+	type: 'Funnel chart'
+	showLegend: boolean
+	display?: FunnelDisplay
```

Comment:
This is really cool, noticed the colors are lacking a little in the vertical view, and wanted to make sure the shape was anticipated
![Screenshot 2024-10-10 at 11 02 50 AM](https://github.com/user-attachments/assets/95317ba2-a361-44ba-9efc-0bc41f403ec0)

Reply:
having chatted w. julian, we're going to hide this view for now and instead allow bar vs. line graph for the funnel